app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java

```diff
@@ -113,8 +115,6 @@ public Fragment getItem(int position) {
                         return ExperimentDocFragment.newInstance("L_Inverting.md");
                     case "Non Inverting Op-Amp":
                         return ExperimentDocFragment.newInstance("L_NonInverting.md");
-                    case "Precision Rectifier":
```

Comment:
why have you removed `precision rectifier` and `human body resistance` experiment calls?